### PR TITLE
The "use strict", minor performance enhancements (the "for" statement), fix typo / style cleanup

### PR DESCRIPTION
**Pull request by janekptacijarabaci**
The suggestion (for example).

"use strict" - in Browser Console:

### CODE REVIEW

File: content/scriptprefs.js [hunk 1]
```diff
@@ -57,11 +57,11 @@ window.addEventListener('load', function() {
 }, false);
 
 function onDialogAccept() {
-  gScript.includes = gScriptIncludesEl.pages;
+  // gScript.includes = gScriptIncludesEl.pages;
```
Comment (arantius) [inline content/scriptprefs.js]: Why these three?

Comment (arantius) [inline content/scriptprefs.js]: Oh; of course.  You can't edit these fields so we shouldn't need to read their values.  Yes?

Comment (janekptacijarabaci) [inline content/scriptprefs.js]: Yes. I'm sorry, I didn't write the details.

**Comment by janekptacijarabaci** [comment]
Merged
https://github.com/greasemonkey/greasemonkey/commit/83ddd7236764539e2ee2c5d427cefad9882a9dc6
https://github.com/greasemonkey/greasemonkey/commit/213b1e6856e5409528c2900cfa541800cff0b764
https://github.com/greasemonkey/greasemonkey/commit/0d569dbe99c5de4510cae45d70efe2e49f5519cc

**Pull request closed without merging**